backend/ventserver/simulation/alarm_mute.py

```diff
@@ -0,0 +1,50 @@
+"""alarm muting request and response"""
+
+import typing
+from typing import Dict, Mapping, Optional
+
+import attr
+
+import betterproto
+
+from ventserver.protocols.backend import states
+from ventserver.protocols.protobuf import mcu_pb
+
+def transform(
+    active_alarm_ids: mcu_pb.ActiveLogEvents,
+    request: mcu_pb.AlarmMuteRequest,
+    response: mcu_pb.AlarmMute
+) -> None:
+    """Mute the current active alarm"""
+    if active_alarm_ids is None:
```

Comment:
`if active_alarm_ids is None` does not catch the case where active_alarm_ids is a message with field `active_alarm_ids.id == []`. Thus, this branch will not be entered in certain situations where you want to enter this branch.

Since the firmware doesn't perform this check, and since there might be some use cases where someone would want to preemptively disable alarms for 2 minutes (e.g. they're about to temporarily disconnect the patient to perform a procedure and don't want alarms during that time), maybe we should just remove this check entirely.